resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/Config.scala

```diff
@@ -90,6 +90,14 @@ private[spark] object Config extends Logging {
       .toSequence
       .createWithDefault(Nil)
 
+  val CONFIG_MAP_MAXSIZE =
+    ConfigBuilder("spark.kubernetes.configMap.maxsize")
+    .doc("Max size limit for a config map. This may be configurable as per" +
+      " https://etcd.io/docs/v3.4.0/dev-guide/limit/ in the future.")
+    .version("3.1.0")
```

Comment:
"Max size limit for a config map. This is configurable as per https://etcd.io/docs/v3.4.0/dev-guide/limit/ on k8s server end."

So may be we should keep it configurable too.